automation/services/watchdog/metrics.py

```diff
@@ -117,9 +117,6 @@ def get_chain_id(v1, namespace):
     try:
       resp = util.exec_on_pod(v1, namespace, pod_name, container_name, 'mina client status --json')      
       resp = resp.strip()
-      if resp[0] != '{':
```

Comment:
This change seems unrelated. Can you remove it?

Reply:
It's related, since it was a workaround due to this :)

Reply:
But happy to remove.

Reply:
It still seems unrelated, but please feel free to explain further.

Reply:
Line 121 says it: `#first line could be 'Using password from environment variable MINA_PRIVKEY_PASS'`

Reply:
But you're right, maybe best to not introduce this change for now.

Reply:
Alright, removed it!